modules/get-started/pages/licensing/overview.adoc

```diff
@@ -31,27 +31,80 @@ The Enterprise Edition is licensed with the https://github.com/redpanda-data/red
+
+[cols="1,2,2"]
+|===
+| Feature | Description | Restrictions Without Valid License
+
+| xref:manage:audit-logging.adoc[Audit Logging]
+| Records detailed logs of cluster activities for compliance and monitoring.
+| Read access to the audit log topic is denied, but logging continues.
+
+| xref:manage:cluster-maintenance/continuous-data-balancing.adoc[Continuous Data Balancing]
+| Automatically balances partitions across a cluster to optimize resource use and performance.
+| Continuous balancing is disabled, reverting to the default `node_add` setting that balances partitions only after a broker is added to the cluster.
+
+| xref:manage:cluster-maintenance/cluster-balancing.adoc#intra-broker-partition-balancing[Continuous Intra-Broker Partition Balancing (`core_balancing_continuous`)]
+| Balances partition replicas across CPU cores in an individual broker to optimize disk space usage.
+| Continuous Intra-Broker Partition Balancing is disabled.
+
+| xref:manage:security/fips-compliance.adoc[FIPS Compliance]
+| Enables compliance with FIPS security standards for cryptography.
+| None
+
+| xref:manage:security/authentication.adoc#kerberos[Kerberos authentication]
+| Provides secure Kerberos-based authentication.
+| None
 
-include::get-started:partial$licensing/enterprise-features.adoc[tag=self-managed]
+| xref:manage:security/authentication.adoc#kerberos[Leader Pinning]
+| Specifies the set of availability zones where the leaders of partitions of a given topic should be located.
+| Leader Pinning is disabled on all topics.
+
+| xref:manage:security/authentication.adoc#oidc[OAUTHBEARER/OIDC authentication]
+| Allows for OAUTHBEARER and OpenID Connect (OIDC) authentication.
+| None
+
+| xref:manage:remote-read-replicas.adoc[Remote Read Replicas]
+| Enables remote clusters to read data stored in object storage for disaster recovery.
+| Remote Read Replica topics cannot be created or modified.
+
+| xref:manage:security/authorization/rbac.adoc[Role-Based Access Control (RBAC)]
+| Manages user roles and permissions within the cluster.
+| Roles and ACLs associated with roles cannot be created or modified. Role deletion is allowed.
+
+| xref:manage:schema-reg/schema-id-validation.adoc[Server-Side Schema ID Validation]
+| Validates schema IDs server-side to ensure schema compatibility. With schema ID validation, records associated with unregistered schemas are detected and dropped by a broker rather than a consumer.
+| Topics with schema validation settings cannot be created or modified.
+
+| xref:manage:tiered-storage.adoc[Tiered Storage]
+| Enables storage of data in cloud object storage for long-term retention and retrieval.
+| Topics cannot be created or modified to enable Tiered Storage features. Additional partitions cannot be added to topics that have any Tiered Storage properties enabled.
+
```

Comment:
iceberg will be an enterprise feature too? Should we include it here?

Reply:
When the feature is ready/documented 👍

Reply:
correct - we decided against any sanctioning on this while it's in public beta